### PR TITLE
Merak Network Document

### PR DESCRIPTION
**Pull request by yanmo96**

### CODE REVIEW

File: docs/components/merak_network.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+- Initial IDs:  
+    ```
+    {
+        "tenantIds": [],
+        "<tenantIds>_projectIds": [],
+        "projectIds": [],
+        "<projectId>_vpc_ids": [],
+        "<vpc_ids>_router_ids": [],
+        "<vpc_ids>_subnet_ids": []
```
Comment (cj-chung) [inline docs/components/merak_network.md:70]: @yanmo96 I am not quit understand why you plan combine two different IDs into one field?

File: docs/components/merak_network.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+___  
+
+- As for the first version of Merak Network component, it will mainly setup virtual network with [Alcor](https://github.com/futurewei-cloud/alcor).  
+- To enable Merak Network have the ability to work with different SDN platform, such as [Alcor](https://github.com/futurewei-cloud/alcor). For now we will have different functions within the code to do different SDN network setup, later may switch to use different micro service do different SDN.  
+- Once received request from the Merak Scenario Manager, Merak Network will start to do its job. Making VPC, Security Group, Router, Subnet; and attach them together.  
+- After setup the virtual network, all information related to future use will be saved into DB. Currently planning on using the mongoDB, the schema will be show in the below section.  
+
+___  
+
+Merak Network will have couple of functions:
```
Comment (cj-chung) [inline docs/components/merak_network.md:34]: @yanmo96 Can you layout the workflow for these functions? You can use a block to represent a process and draw a line between processes to show the workflow. You can refer to Zhen's design.

Comment (yanmo96) [reply docs/components/merak_network.md:34]: Hi @cj-chung is the following graph ok? Thanks
https://raw.githubusercontent.com/yanmo96/merak-1/network_doc/docs/images/MerakNetwork_v2.drawio.svg

File: docs/components/merak_network.md (hunk 1 above)
Comment (yanmo96) [reply docs/components/merak_network.md:70]: I was thinking we can list all `vac_id` by given `project_id`

File: docs/components/merak_network.md (hunk 2 above)
Comment (yanmo96) [reply docs/components/merak_network.md:34]: ok

Comment (yanmo96) [reply docs/components/merak_network.md:34]: Hi @cj-chung, I have update the image and DB according to our meeting earlier. Please let me know if is ok, Thanks

**Review by cj-chung — APPROVED** [review]
LGTM for now, @yanmo96 please update the design doc while you're coding. The design here is still very high level.

**Pull request merged**